graspologic/embed/base.py

```diff
@@ -165,7 +175,7 @@ def fit_transform(self, graph, y=None):
             If directed, ``concat`` is True then concatenate latent matrices on axis 1, shape(n_vertices, 2*n_components).
             If directed, ``concat`` is False then tuple of the latent matrices. Each of shape (n_vertices, n_components).
         """
-        return self._fit_transform(graph)
+        return self._fit_transform(graph, *args, **kwargs)
```

Comment:
Why do you need this function if it is only calling self._fit_transform ?

Reply:
the whole function? not sure, I wasn't the one who designed the base class like that, I think it was @j1c two years ago. I'd guess it's to make the `fit_transform` code private? but unclear to me

the `*args, **kwargs` addition is because the `fit` function in CASE takes in a covariate matrix as well as an adjacency matrix, so `fit_transform` needs to be able to take in extra variables now.

Reply:
ok that makes sense. I think maybe making a comment about that within the function may be helpful?